telegram-bot/meetup.go

```diff
@@ -77,8 +102,13 @@ func addmeetup(ID int64, msgtext string, client mongo.Client) {
 			log.Fatal(err)
 		}
 		bot.Send(tbot.NewMessage(ID, "Meetup added successfully."))
+		nextmeetup(ID, client)
+		remindTime := fmt.Sprintf("%d:%d:00", hh-1, min) //making a formatted string from time to pass it to s.At()
+		s1 := gocron.NewScheduler(time.Local)
+		s1.Every(1).Day().At(remindTime).Do(reminder, ID, client, s1) //calls the reminder function
+		s1.Start()                                                    //starts the scheduler
 	} else {
-		bot.Send(tbot.NewMessage(ID, "Please provide the details of meetup in this format - /addmeetup <Title> <Date> <Venue>"))
+		bot.Send(tbot.NewMessage(ID, "Please provide the details of meetup in this format - /addmeetup <Title> <DD MM YY Hr Min> <Venue> \nTIME IN 24Hr FORMAT"))
```

Comment:
Why do we need time in 24 Hour format? Does the cronjob package needs it in that way?

Reply:
I'm not sure about gocron needing it that way but as of now func `At` only reads 24hr format [documentation](https://github.com/jasonlvhit/gocron/blob/master/job.go#L120). 
So yeah we have to use it. Will it be a problem? :confused: